src/content/docs/en/basics/astro-syntax.mdx

```diff
@@ -173,3 +173,140 @@ In Astro, you can use standard HTML comments or JavaScript-style comments.
 :::caution
 HTML-style comments will be included in browser DOM, while JS ones will be skipped. To leave TODO messages or other development-only explanations, you may wish to use JavaScript-style comments instead.
 :::
+
+
+## Component utility functions
+
+### `Astro.slots`
+
+`Astro.slots` contains utility functions for modifying an Astro component's slotted children.
+
+#### `Astro.slots.has()`
+
+<p>
+
+**Type:** `(slotName: string) => boolean`
+</p>
+
+You can check whether content for a specific slot name exists with `Astro.slots.has()`. This can be useful when you want to wrap slot contents, but only want to render the wrapper elements when the slot is being used.
```

Comment:
```suggestion
You can check whether content for a specific slot name exists with `Astro.slots.has()`. This can be useful when you want to wrap slot contents but only want to render the wrapper elements when the slot is being used.
```